store/extensions/UserAdderReborn/extension.json

```
@@ -0,0 +1,45 @@
+{
+  "title": "UserAdderReborn",
+  "description": "Reborned Version of the UserAdder, it will add all users upon entering a room. Thanks schweppes0x",
+  "authors": [
+    {
+      "name": "Thauan",
+      "hotel": ".com.br",
+      "username": "Erotico",
+      "discord": "thauanvargas"
+    },
+    {
+      "name": "schweppes0x",
+      "hotel": ".nl",
+      "username": "-MrTn-",
+      "discord": "Itstonyownss#0683"
+    }
+  ],
+  "version": "1.0",
+  "categories": ["Usability"],
+
+  "source": "https://github.com/thauanvargas/UserAdder",
+  "readme": "https://github.com/thauanvargas/UserAdder/blob/main/README.md",
```

This repository doesn't exist you probably meant this:
```suggestion
  "source": "https://github.com/thauanvargas/UserAdderReborn",
  "readme": "https://github.com/thauanvargas/UserAdderReborn/blob/main/README.md",
```